other/magicdiamondpattern.py

```diff
@@ -0,0 +1,53 @@
+# Python program for generating diamond pattern in python 3.7+
+
+# Function to print upper half of diamond (pyramid)
+def floyd(n):
+	'''
+	Parameters: 
+    n : size of pattern 
+	'''
+	for i in range(0, n):
+		for j in range(0, n-i-1): # printing spaces
+			print(" ", end = "")
+		for k in range(0, i + 1): # printing stars 
+			print("* ", end = "")
+		print()
+
+
+# Function to print lower half of diamond (pyramid)
+def reverse_floyd(n):
+	'''
+	Parameters: 
+    n : size of pattern 
+	'''
+	for i in range(n, 0, -1):
+		for j in range(i, 0, -1):  # printing stars 
+			print("* ", end = "")
+		print()
+		for k in range(n-i+1, 0, -1):  # printing spaces 
+			print(" ", end = "")
+
+# Function to print complete diamond pattern of "*"
+def pretty_print(n):
+	'''
+	Parameters: 
+    n : size of pattern 
+	'''
+	if n <= 0:
+		print("       ...       ....        nothing printing :(")
+		return 
+	floyd(n)   # upper half 
+	reverse_floyd(n) # lower half 
+
+
+if __name__ == "__main__":
+	print("| /\ | |- |  |-  |--| |\  /| |-")
+	print("|/  \| |- |_ |_  |__| | \/ | |_")

```

Comment:
Can you please make these two lines __r"strings"__ instead of just __"strings""__ to avoid these warnings: https://travis-ci.org/TheAlgorithms/Python/builds/596096974#L514

Reply:
oh, okay sure.
thanks for reviewing.

Reply:
Done